web/src/features/charts/bar-breakdown/utils.ts

```diff
@@ -145,13 +153,14 @@ export interface ExchangeDataType {
   mode: ZoneKey; // TODO: Weird that this is called "mode"
   gCo2eqPerkWh: number;
   tCo2eqPerMin: number;
+  exchangeCapacityRange: number[];
 }
 export const getExchangeData = (
   data: ZoneDetail,
-  exchangeKeys: string[],
+  exchangeKeys: ZoneKey[],
   electricityMixMode: Mode
 ): ExchangeDataType[] =>
-  exchangeKeys.map((mode) => {
+  exchangeKeys.map((mode: ZoneKey) => {
     // Power in MW
     const exchange = (data.exchange || {})[mode];
```

Comment:
```suggestion
    const exchange = data.exchange?.[mode];
```
Let's clean this up while we are here 🙂